src/components/FixedProgress.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import '../styles/fixedProgress'
+
+export default class Notification extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.state = {
+      progress: 0.000001,
+      id: null
+    }
+  }
+  componentDidMount () {
+    const {interval} = this.props
+    const id = window.setInterval(() => {
```

Comment:
I think that we can use the global `setInterval` and don't need the `window` here.